public/index.html

```diff
@@ -2,7 +2,10 @@
 <html lang="en">
   <head>
     <meta charset="utf-8" />
-    <link rel="icon" href="%PUBLIC_URL%/favicon.ico" />
+    <link
+      rel="icon"
+      href="https://cleanjuice.com/wp-content/uploads/2020/04/cropped-favicon-32x32.png"
```

Comment:
This file must be in the public folder, because if there is a change in the hosted server it will not be at risk of breaking.